ivy_tests/test_ivy/helpers/testing_helpers.py

```diff
@@ -160,7 +161,7 @@ def _get_supported_devices_dtypes(fn_name: str, fn_module: str):
 
 
 def handle_test(
-    *, fn_tree: str, ground_truth_backend: str = "tensorflow", **_given_kwargs
+    *, fn_tree: str, ground_truth_backend: str = "numpy", **_given_kwargs
```

Comment:
I see that the default ground truth has been reset to numpy. This willl require a reconsideration for tests with `test_gradients=True` which may need to have to manually set tensorflow/jax/torch for their `test_function` call. This will lead to a dependency on tensorflow/jax etc in the [following line](https://github.com/unifyai/ivy/blob/8010806d72537c94b1b88eab5530739c80c4ecbc/ivy_tests/test_ivy/helpers/function_testing.py#L267) in test_function unless there is way to bypass such tests. Have you tried running any such tests. You can try `test_acosh` which will should fail on this branch for examples where gradient test is run.

Same comments can be made for the update in `handle_method` although it had some other updates that Mustafa was working on I suppose.

Reply:
Ok I'll revert back to using to setting the `ground_truth_backend` to tensorflow and then leave it at the discretion of the developer to install tensorflow if they wish to use these tests, as it will be clear from the error received when running the test.

Reply:
Can we instead not have a script which is run at import time to scan the available packages. If tensorflow/jax/torch is not found in the environment then gt should be set to numpy for all tests globally with a warning raised that gradient tests are unreliable with current environment?🙂

Reply:
Yeah thank you, we can quite easily actually; I've implemented that in my latest commit. If the user has tensorflow installed, it first uses that. If they don't have tensorflow and they have torch, then it is set to torch. If they have only jax and numpy, it is set to jax. Otherwise it is set to numpy. Everything should work now 🙂 Let me know what you think.